<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>takeout_cook</class>
 <widget class="QDialog" name="takeout_cook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox::up-button {
width:0px;
} 
QDoubleSpinBox::down-button {
width:0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>frame_detail_show</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1">
      <item row="5" column="1">
       <widget class="QFrame" name="frame_2">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_sendman"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_sendman">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>送餐员</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_desc">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>外卖号</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="lineEdit_orderman"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>抹零</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_sendman">
        <property name="text">
         <string>送餐员</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_sn">
        <property name="styleSheet">
         <string notr="true">color:#ff546b;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_tel">
        <property name="text">
         <string>电话</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_orderman">
        <property name="text">
         <string>订餐人</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>税收</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_sendtime">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_sendaddress">
        <property name="text">
         <string>地址</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>应收</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>赠送</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="lineEdit_sendaddress"/>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="lineEdit_tel"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_operid"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>消费</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>操作员</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>折扣</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="lds_doublespinbox_lineedit" name="lineEdit_consum"/>
      </item>
      <item row="2" column="1">
       <widget class="lds_doublespinbox_lineedit" name="lineEdit_discount"/>
      </item>
      <item row="3" column="1">
       <widget class="lds_doublespinbox_lineedit" name="lineEdit_blotout"/>
      </item>
      <item row="3" column="3">
       <widget class="lds_doublespinbox_lineedit" name="lineEdit_realreceive"/>
      </item>
      <item row="2" column="3">
       <widget class="lds_doublespinbox_lineedit" name="lineEdit_int_rate"/>
      </item>
      <item row="1" column="3">
       <widget class="lds_doublespinbox_lineedit" name="lineEdit_present"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_doublespinbox_lineedit</class>
   <extends>QLineEdit</extends>
   <header>lds_doublespinbox_lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
